sdk/servicebus/azure-servicebus/samples/sync_samples/receive_queue.py

```diff
@@ -20,7 +20,7 @@
 servicebus_client = ServiceBusClient.from_connection_string(conn_str=CONNECTION_STR)
 
 with servicebus_client:
-    receiver = servicebus_client.get_queue_receiver(queue_name=QUEUE_NAME, prefetch=10)
+    receiver = servicebus_client.get_queue_receiver(queue_name=QUEUE_NAME)
```

Comment:
Do we use or discuss prefetch in the README anywhere that we'd want to nix?

Reply:
README doesn't mention "prefetch".

But I have updated the docstring and samples.

Reply:
Echoing from our OOB chat for posterity:

I might just rearrange your stanzas.

 

Prefetch: the maximum <etc etc.> to the service.
This setting is only for advanced performance tuning, Increasing this value will <the rest of your warning here>
Default value is 0, meaning <etc>; in the case of 0 <etc>

(Feel free to ping me whenever you're green on tests and I can OK this)

Reply:
already updated, thanks for the suggestion!